src/java/org/apache/cassandra/db/virtual/SystemViewsKeyspace.java

```diff
@@ -45,6 +50,12 @@ private SystemViewsKeyspace()
                                                               new ThreadPoolRowWalker(),
                                                               Metrics.allThreadPoolMetrics(),
                                                               ThreadPoolRow::new))
+                    .add(CollectionVirtualTableAdapter.create(VIRTUAL_VIEWS,
+                                                              STORAGE_OPERATIONS,
+                                                              "Storage operations currently running or completed for the last 7 days",
```

Comment:
you hardcoded 7 days here not reflecting what a user might set it to

Reply:
Fixed.